pkgs/games/minetest/default.nix

```diff
@@ -29,37 +27,33 @@
 , postgresql
 , hiredis
 , libiconv
-, zlib
-, libXrandr
-, libX11
 , ninja
 , prometheus-cpp
-, mesa
 , OpenGL
 , OpenAL ? openal
 , Carbon
 , Cocoa
-, withTouchSupport ? false
+, Kernel
 , buildClient ? true
 , buildServer ? true
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "minetest";
-  version = "5.8.0";
+  version = "5.9.0";
 
   src = fetchFromGitHub {
     owner = "minetest";
     repo = "minetest";
     rev = finalAttrs.version;
-    hash = "sha256-Oct8nQORSH8PjYs+gHU9QrKObMfapjAlGvycj+AJnOs=";
+    hash = "sha256-cxbiuoD1J3WFoveUgxeR/XXdE7MMR0UEDFleDiaxnsA=";
   };
 
   cmakeFlags = [
     (lib.cmakeBool "BUILD_CLIENT" buildClient)
     (lib.cmakeBool "BUILD_SERVER" buildServer)
     (lib.cmakeBool "ENABLE_PROMETHEUS" buildServer)
-    (lib.cmakeBool "ENABLE_TOUCH" withTouchSupport)
+    (lib.cmakeBool "USE_SDL2" useSDL2)
```

Comment:
`useSDL2` is undefined here

Reply:
Ah I see you added the argument in a later commit. Either squash them, or move this line to the other commit

Reply:
Good catch, this must have happened when I rebased. Fixed now.